public/scripts/power-user.js

```diff
@@ -662,6 +663,8 @@ async function CreateZenSliders(elmnt) {
         sliderID == 'dynatemp_exponent_textgenerationwebui' ||
         sliderID == 'smoothing_curve_textgenerationwebui' ||
         sliderID == 'smoothing_factor_textgenerationwebui') {
```

Comment:
> I think this is causing an error. ") {" => " ||"

That right. It needs to be fixed.
```suggestion
        sliderID == 'smoothing_factor_textgenerationwebui' ||
```

Reply:
Fixed.